cross/dkb-trace/Makefile

```diff
@@ -0,0 +1,19 @@
+PKG_NAME = dkb-trace
+PKG_VERS = 2.2
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://digitalbox.chez.com/sources
```

Comment:
Personalty I prefer a more authoritative source. chez(dot)com is listed as engaged in malware distribution: https://hosts-file.net/default.asp?s=chez.com